### ISSUE TITLE
Optional commas in dependency lists

### ISSUE THREAD

**Issue by hdgarrood** [haskell/cabal#1509]
From [this page](http://hackage.haskell.org/trac/hackage/wiki/Cabal-2):

> Related to above, allow layout for lists of tokens, to make the comma
> optional in lists of deps if they're on separate lines. Allow trailing
> commas?

I'm interested in having a go at implementing this. Has anything already been done? Should I just get stuck in?

**Comment by osa1** [comment]
Does anyone have any news on that? Is the work on new parser done?

**Comment by phadej** [comment]
There is #4971, so you can write:
```
executable test
    -- note: no comma after "base" here
    build-depends:
      , base
      , random
```
to make your build-depends sort and diff friendly.

**Comment by 23Skidoo** [comment]
Do we have a changelog note for this? Otherwise looks like this ticket can be closed as fixed.

**Comment by phadej** [comment]
https://github.com/haskell/cabal/blob/6750de6ae2ba28e748b5094e2d73034dfaaac426/Cabal/changelog#L12

**Comment by 23Skidoo** [comment]
Closing as fixed by #4971.
